src/config/mail/mail.config.ts

```diff
@@ -0,0 +1,7 @@
+import { registerAs } from "@nestjs/config";
+
+export default registerAs("mail", () => ({
+    mailjetApiPublic: process.env.MJ_APIKEY_PUBLIC,
+    mailjetApiPrivate: process.env.MJ_APIKEY_PRIVATE,
+    frontendUrl: process.env.FRONTEND_URL,
```

Comment:
should frontendUrl be in the app config?

Reply:
I didn't put it in the app config because it was only being used for emails and I was trying to group the config into their usage.

Reply:
I think I also use that for OAuth, redirect to frontend
![image](https://github.com/user-attachments/assets/796abb8b-da7a-4dac-a2b9-7ad21faa37c8)

Reply:
ok, I'll work on moving it to appconfig then.

Reply:
I have moved `FrontendUrl` to the appConfig.